File: 2022/day1/star1.cpp
#include <iostream>
#include <fstream>
#include <string>
using namespace std;

int main() {
    string line;
    ifstream input("input.txt");
    int max = 0, cur = 0;
    while(getline(input, line)) {
        if(line.empty()) {
            max = max > cur ? max : cur;
            cur = 0;
        }
        else {
            cur+=stoi(line);
        }
    }
    max = max> cur ? max : cur;
    cout << max << endl;
}

File: 2025/day3/star1.py
total_joltage_output = 0
with open("2025/day3/input.txt") as file:
    for line in file:
        v = list(map(int, list(line.strip())))
        largest_num = max(v[:-1])
        total_joltage_output+=(10*largest_num) + max(v[v.index(largest_num)+1:])

print(total_joltage_output)
